LayoutTests/imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-GC.https.html

```diff
@@ -69,7 +72,7 @@
   destVideo.srcObject = new MediaStream([(await haveTrackEvent).track]);
 
   // Wait for video on the other side.
-  await onVideoChange();
+  await destVideo.play();
```

Comment:
the play promise is resolved when playback can start, that is for example the user interacted with the video. Not when there's a frame or anything showing from playback having started.

Reply:
I'll do a requestVideoFrameCallback check if present instead.

Reply:
Actually no, this makes the test fail in Chrome, I am not sure why.
I'll keep the test like this.